lib/Clam/CfgBuilder.cc

```diff
@@ -1961,6 +2000,36 @@ void CrabIntraBlockBuilder::doMemIntrinsic(MemIntrinsic &I) {
   m_bb.havoc(m_lfac.mkIntVar(32), valueToStr(I));
 }
 
+void CrabIntraBlockBuilder::doGepFatAssign(Instruction &I, unsigned width,
+                                           lin_exp_t offset) {
+  if (!isa<BitCastInst>(I) && !isa<GetElementPtrInst>(I)) {
+    return;
+  }
+  CRAB_LOG("cfg-fat", llvm::errs()
+                          << "Gep: add two extra variable for " << I << "\n");
+  Value *vlhs = llvm::cast<llvm::Value>(&I);
+  Value *vptr = I.getOperand(0);
+  auto it = m_ref_fat_map.find(vptr);
+  if (it == m_ref_fat_map.end()) {
+    CLAM_WARNING("Could not find gep ptr from map " << *vptr << "\n at" << I);
```

Comment:
I would trigger an error here (`CLAM_ERROR`). Clam already produces many warnings so it's easy to miss some